platform/admin/core/admin-core-logviewer/src/main/webapp/js/components/log-entry/log-entry.js

```diff
@@ -16,14 +16,17 @@
 import React from 'react'
 import moment from 'moment'
 
+import './log-entry.less'
+
 import levels from '../../levels'
+import * as actions from '../../actions'
 
 const format = (time) => {
   return moment(time).format('D MMM YYYY, HH:mm:ss')
 }
 
 // log entry to display
-export default ({ entry, marks }) => {
+export default ({ entry, marks, expandedHash, dispatch }) => {
```

Comment:
Can we give this function a clearer name, e.g., `displayEntryWithHighlights` or something?